src/renderer/shared/i18n/locales/en.json

```diff
@@ -154,9 +154,9 @@
     "multisigStep": "Step { step }.",
     "nameDescription": "Give a name to your multisig account",
     "nameLabel": "Account",
-    "nameNetworkDescription": "Select the network and name for your multisig wallet",
+    "nameNetworkDescription": "Select the network and name for your multisig",
```

Comment:
according to [layouts](https://www.figma.com/design/HgFuj8QZd0xqEbkG10DIfs/Flexible-multisig-%7C-Milestone-4?node-id=815-41265&t=T99kW8RMsCtXdMqz-1) the string is: `Step 1. Select the network and the name of your multisig.`